drydock/templates/kustomized/global-stack/kustomization.yml

```diff
@@ -0,0 +1,19 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: monitoring
```

Comment:
should we change this namespace? @MoisesGSalas

Reply:
Probably use a variable.